docs/utilities/idempotency.md

```diff
@@ -55,7 +55,7 @@ times with the same parameters**. This makes idempotent operations safe to retry
                      <aspectLibraries>
                          <aspectLibrary>
                              <groupId>software.amazon.lambda</groupId>
-                             <artifactId>powertools-idempotency-dynamodb</artifactId>
+                             <artifactId>powertools-idempotency-core</artifactId>
```

Comment:
Is this intended to change? Idk if you are publishing a new artifact, but I can't find this in maven repo.

Reply:
Forget about this, resolving!